app/(funnel)/components/ScoreInput/ScoreInput.tsx

```
@@ -0,0 +1,76 @@
+    const measured = hiddenSpanRef.current.offsetWidth;
+    const minWidthPx = 48;
+    setScoreWidthPx(Math.max(measured, minWidthPx));
+  };
+
+  const handleScoreChange = (e: ChangeEvent<HTMLInputElement>) => {
+    let value = e.target.value.replace(/[^\d.]/g, ''); // 숫자와 점만 허용
+
+    // 첫 번째 숫자가 4보다 크면 4로 제한
+    if (value.length === 1 && parseInt(value) > 4) {
+      value = '4';
+    }
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)